doc/rules/array_notation/multiline_long_array.rst

```diff
@@ -0,0 +1,54 @@
+=============================
+Rule ``multiline_long_array``
+=============================
+
+A single-line array should be broken into multiple lines if it exceeds
+configured limit.
+
+Configuration
+-------------
+
+``max_length``
```

Comment:
Isn't that like common practice in the whole universe that the length of an array is the number of elements? Since `max_length` here is not can we have a good description of what is it?